### ISSUE TITLE
undefined method `next_time' for nil:NilClass

### ISSUE THREAD

**Issue by jankeesvw** [sidekiq-scheduler/sidekiq-scheduler#445]
I am updating the gems in our Gemfile, and I'm now running into an error on this line:

https://github.com/sidekiq-scheduler/sidekiq-scheduler/blob/87dc745d7f82a6ce89fea6c3a5eb5ffae223532b/lib/sidekiq-scheduler/scheduler.rb#L132

Still figuring out what's going on, but I am already creating this issue so other people can find it too.

**Comment by jankeesvw** [comment]
I haven't found the time to fully dig into it. I created the issue mostly so people can find this when they run into this. I imagine that it's our combination of gems. I did a bundle update upgraded to Sidekiq 6.5.9. 

I'm going to have to figure this out soon, if I find something I'll post it here.

**Comment by mauricebolhuis** [comment]
I encountered the same error (using sidekiq 7.2.2 and sidekiq-scheduler 5.0.3). Starting of today I got the following error while attempting to start Sidekiq:

```
11:36:07 sidekiq.1 | 2024-05-01T09:36:07.190Z pid=515927 tid=axvj INFO: Scheduling foo {"class"=>"FooJob", "at"=>"2024/04/30", "enabled"=>false, "queue"=>"default"}
11:36:07 sidekiq.1 | 2024-05-01T09:36:07.191Z pid=515927 tid=axvj WARN: {"context":"Exception during Sidekiq lifecycle event.","event":"startup"}
11:36:07 sidekiq.1 | 2024-05-01T09:36:07.191Z pid=515927 tid=axvj WARN: NoMethodError: undefined method `next_time' for nil
11:36:07 sidekiq.1 | 2024-05-01T09:36:07.191Z pid=515927 tid=axvj WARN: 
11:36:07 sidekiq.1 | undefined method `next_time' for nil
11:36:07 sidekiq.1 | /home/maurice/.rbenv/versions/3.3.0/lib/ruby/gems/3.3.0/gems/sidekiq-scheduler-5.0.3/lib/sidekiq-scheduler/scheduler.rb:133:in `block in load_schedule_job'
11:36:07 sidekiq.1 | /home/maurice/.rbenv/versions/3.3.0/lib/ruby/gems/3.3.0/gems/sidekiq-scheduler-5.0.3/lib/sidekiq-scheduler/scheduler.rb:123:in `each'
11:36:07 sidekiq.1 | /home/maurice/.rbenv/versions/3.3.0/lib/ruby/gems/3.3.0/gems/sidekiq-scheduler-5.0.3/lib/sidekiq-scheduler/scheduler.rb:123:in `load_schedule_job'
11:36:07 sidekiq.1 | /home/maurice/.rbenv/versions/3.3.0/lib/ruby/gems/3.3.0/gems/sidekiq-scheduler-5.0.3/lib/sidekiq-scheduler/scheduler.rb:101:in `block in load_schedule!'
11:36:07 sidekiq.1 | /home/maurice/.rbenv/versions/3.3.0/lib/ruby/gems/3.3.0/gems/sidekiq-scheduler-5.0.3/lib/sidekiq-scheduler/scheduler.rb:99:in `each'
11:36:07 sidekiq.1 | /home/maurice/.rbenv/versions/3.3.0/lib/ruby/gems/3.3.0/gems/sidekiq-scheduler-5.0.3/lib/sidekiq-scheduler/scheduler.rb:99:in `load_schedule!'
11:36:07 sidekiq.1 | /home/maurice/.rbenv/versions/3.3.0/lib/ruby/gems/3.3.0/gems/sidekiq-scheduler-5.0.3/lib/sidekiq-scheduler/manager.rb:24:in `start'
11:36:07 sidekiq.1 | /home/maurice/.rbenv/versions/3.3.0/lib/ruby/gems/3.3.0/gems/sidekiq-scheduler-5.0.3/lib/sidekiq-scheduler/sidekiq_adapter.rb:41:in `start_schedule_manager'
11:36:07 sidekiq.1 | /home/maurice/.rbenv/versions/3.3.0/lib/ruby/gems/3.3.0/gems/sidekiq-scheduler-5.0.3/lib/sidekiq-scheduler.rb:21:in `block (2 levels) in <main>'
```

I have a job that has been scheduled using an `at` schedule type:

```sidekiq.yml
:scheduler:
  :schedule:
    foo:
      class: FooJob
      at: '2024/04/30'
      enabled: false
```

It seems like the error occurs when a job has been scheduled using an `at` rule with a date in the past. I have created a pull request to resolve this issue: https://github.com/sidekiq-scheduler/sidekiq-scheduler/pull/466

**Comment by marcelolx** [comment]
Thank you so much @mauricebolhuis! I will look into this issue next week along with other PRs I have on my list! I really appreciate the PR 🙏

**Comment by marcelolx** [comment]
Fixed by #466 466
